### PR TITLE
clarify info re: IP addresses and Network Broker

### PR DESCRIPTION
**Pull request by khorne3**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR

### CODE REVIEW

File: docs/deployment/deployment-checklist.md [hunk 1]
```diff
@@ -314,7 +314,11 @@ Semgrep requires the following permissions (scopes) to enable the authentication
 
 <IpAddresses />
 
-Alternatively, you can use the [Semgrep Network Broker](/docs/semgrep-ci/network-broker) to facilitate secure access with Semgrep instead of allowlisting these IP addresses.
+#### Allowlists when using Semgrep Network Broker
+
+The [Semgrep Network Broker](/docs/semgrep-ci/network-broker) facilities secure access with Semgrep, and its use can replace the allowlisting of the IP addresses required for ingress. The Network Broker, however, only facilitates requests from Semgrep to your network and *doesn't* assist with requests originating from your network, including those from your network to Semgrep.
+
+In other words, the only address you would have to allow inbound is `wireguard.semgrep.dev` on UDP port `51820`, but depending on how restrictive your network is, you may need to allowlist all of the egress IP addresses provided in [IP addresses](#ip-addresses).
```
Comment (s-santillan) [inline docs/deployment/deployment-checklist.md:321]:
```suggestion
In other words, the only address you would have to allow inbound is `wireguard.semgrep.dev` on UDP port `51820`, but depending on how restrictive your network is, you may need to add to your allowlist all of the egress IP addresses provided in [IP addresses](#ip-addresses).
```

File: docs/deployment/deployment-checklist.md [hunk 2]
```diff
@@ -318,7 +318,7 @@ Semgrep requires the following permissions (scopes) to enable the authentication
 
 The [Semgrep Network Broker](/docs/semgrep-ci/network-broker) facilities secure access with Semgrep, and its use can replace the allowlisting of the IP addresses required for ingress. The Network Broker, however, only facilitates requests from Semgrep to your network and *doesn't* assist with requests originating from your network, including those from your network to Semgrep.
 
-In other words, the only address you would have to allow inbound is `wireguard.semgrep.dev` on UDP port `51820`, but depending on how restrictive your network is, you may need to allowlist all of the egress IP addresses provided in [IP addresses](#ip-addresses).
+In other words, the only address you would have to allow inbound is `wireguard.semgrep.dev` on UDP port `51820`, but depending on how restrictive your network is, you may need to add to your allowlist all of the egress IP addresses provided in [IP addresses](#ip-addresses).
```
Comment (armchairlinguist) [inline docs/deployment/deployment-checklist.md:321]:
```suggestion
In other words, the only address you would have to allow inbound is `wireguard.semgrep.dev` on UDP port `51820`, but depending on how restrictive your network is, you may need to modify your allowlist all to include the egress IP addresses provided in [IP addresses](#ip-addresses).
```

**Review by armchairlinguist — COMMENTED** [review]
just a light wording suggestion to avoid having both heavy participial and noun phrases.

**Review by s-santillan — APPROVED** [review]
Thank you!

**Pull request merged**